doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -323,12 +323,11 @@ at the time of use of the notation.
    the parser is not modified at all.
 
    To a given notation string and scope can be attached at most one
```

Comment:
Github won't let me highlight the entire paragraph :-(

It's hard to make sense of it (especially starting with the infinitive and using the passive voice "can be").  Here's my best guess at what's intended (probably not correct):

> Notations marked with `only printing` can have multiple associated interpretations; each interpretation is associated with a particular scope.  If `only printing` is not specified, then there can only be a single associated interpretation associated with all scopes.  Accordingly ...

IIUC some of this should be appear in the description of several of the notation commands.

I don't like the term "interpretation".  It suggests the result of some unspecified process (very general) but IIUC it is just the notation string and it's just for parsing/printing.

IMHO we may need to iterate on the description here.  The wording is too vague/undefined.